### PR TITLE
[DoctrineBridge] run tests against doctrine/persistence 2.1

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
We can revert this once doctrine/orm#8651 is merged.

**Review by derrabus — APPROVED** [review]
This should really just be a temporary solution to unblock the CI.

**Comment by Nyholm** [comment]
Thank you. I agree that this is a temporary solution until doctrine/orm is updated.

**Pull request merged**
